Sources/SQLiteDriver.swift

```diff
@@ -2,7 +2,15 @@ import Fluent
 
 public class SQLiteDriver: Fluent.Driver {
 
-	let database = SQLite()
+	var database = SQLite()
+
+	public init() {
+		database.initWithPath("Database/main.sqlite")
+	}
+
+	public init(path: String){
+		database.initWithPath(path)
```

Comment:
@tannernelson what do you think about enforcing that the database should be inside the `Database/` direcotry?